README.md

```diff
@@ -404,18 +404,25 @@ You can read about [getting started](https://julialang.org/manual/getting-starte
 
 The following distributions include julia, but the versions may be out of date due to rapid development:
 
+<a href="https://repology.org/metapackage/julia">
+    <img src="https://repology.org/badge/vertical-allrepos/julia.svg" alt="Packaging status" align="right">
```

Comment:
Yes, it's showing the lastest version available for a specific repo. Showing all of them would make it just too big, and availability of legacy versions are not really interesting for most users.